### ISSUE TITLE
Ripple Content

### ISSUE THREAD

**Issue by peteratticusberg** [peteratticusberg/orc#60]
Mike Nard [on Reddit](https://www.reddit.com/r/Ripple/comments/82hget/a_simple_explanation_of_ripple_for_the_non_and/):

Ok i thank you for TRYING to make it easier for new comers, but most of your info and even understand of what xrp and xrapid does is off. Ill Try to make it easy to explain exactly what, but theres a lot.

To the extent Ripple is used to exchange different assets (e.g. gold for euros), it relies on parties settling transaction outside of RippleNet (src). This makes this application of Ripple difficult to apply outside a network of trusted and sophisticated actors (viz. outside of the banking world).

--This is not true It does not rely on parties to settle transactions outside of ripple net, if a company is using xcurrent they have prefunded accounts, which xcurrent puts the transaction amount on hold from each end ( sending and receiving) before the transactions is made so it CANT FAIL, Therefore this whole section is invalid. https://ripple.com/files/ripple_product_overview.pdf page 7 will show how accounts are prefunded and settle within the network.

Because admission into the network that secures and powers Ripple is gated by the large organizations that currently run Ripple nodes, Ripple also faces resistance from actors within the cryptocurrency world who favor more open and more decentralized models.

--There is no admission to the network at all. The network is open source, to run a node it is permission-less,open key, and anybody can be a node instantly upon setting one up. These nodes also get to vote for changes. Here is evidence of these statements, and it is where one can begin to set up a node https://ripple.com/build/rippled-setup/ ALSO Ripple is completely decentralized( 600 nodes and counting), just as decentralized as any other crypto, and it will be the most decentralized within the next 6 months because the addition of more geographically diverse nodes. I could write the 4 paragraphs reasons and 3 ares of decentralization with sources but this article would be 25x longer. Here is node list.https://xrpcharts.ripple.co...

note that Ripple is particularly well suited to facilitate the exchange of one asset for another, e.g. USD for EUR, gold bullion for XRP, or BTC for JPY. Also note that for all such exchanges, all RippleNet does is update a balance sheet shared between two large financial institutions which describes how much of a given asset each owes the other. It is up to the financial insitutions to actually pay each other what they are owed (src) in USD, BTC, gold bullion, etc. XRP is the one exception to this. RippleNet can transfer XRP instantly.

-- This is not all ripple net does at all, that is not even the tip of the ice burg. EVEN ASSETS LIKE GOLD AND OIL COULD BE USED ON XCURRENT. Ripple net can be used as a liquidity source, an intermediary assets, a transaction facilitator, an inventory tracker, identification, healthcare. The possibilities are endless

Also They either have prefunded accounts for xcurrent( it is not up to them to settle after), or they instantly purchase xrp through xrapid, and the xrp is sold for the currencey of where they are sending to ( xrapid does all this in a 4-10 second transaction end to end SETTELED).

I think if your going to explain ripple labs and xrp in an easy way to someone, the most important part would be dicussing the difference between ripple labs, and xrp. Also the difference and purpose for xrapid and xcurrent.

**Comment by peteratticusberg** [comment]
Me [on Reddit](https://www.reddit.com/r/Ripple/comments/82hget/a_simple_explanation_of_ripple_for_the_non_and/): 

Thanks for this, this is definitely helpful. I've copy pasted this correspondence into an issue on the github repo, if you're interested in responding would appreciate if you could do it there.

So it sounds like I make three claims that are problematic:

1) Ripple relies on parties transacting on RippleNet to settle outside of RippleNet

2) "admission into the network that secures and powers Ripple is gated by the large organizations that currently run Ripple nodes"

3) "Ripple is particularly well suited to facilitate the exchange of one asset for another, e.g. USD for EUR, gold bullion for XRP, or BTC for JPY"

1: As far as this is concerned, the source I cite (ripple.com's gateway documentation) for that claim states the following:

All assets in the XRP Ledger, except for the native cryptocurrency XRP, are represented as issuances, which are digital balances that represent currency or assets of value held by an issuer. Within the XRP Ledger, counterparties can send and trade issuances without requiring intervention from the issuer. Typically, a gateway sends issuances to customers when it receives money in systems and ledgers outside the XRP Ledger, and promises to send money to customers in outside systems in exchange for being repaid in issuances in the XRP Ledger. Issuances get their value from a gateway's agreement to honor the obligation that the issuances represent. No computer system can force an XRP Ledger gateway to honor that obligation.

This to me sounds consistent with my description of how this process works. I get that the XRP ledger != RippleNet and that Ripple provides additional tools beyond the XRP ledger. It sounds like in this case xCurrent specifically allows for the creation of prefunded accounts in ways that decrease the amount of trust required between transacting parties. What I wanted to convey is that XRP cannot transport USD, EUR, gold, etc. between parties in an on-chain way (nor could any technology, of course). Could you think of a way to rewrite that paragraph that preserves that sentiment while also more accurately conveying the capabilities of RippleNet? If not I can rework it.

2: I think this is just an issue of semantics. I agree that anybody can run a Ripple node. But that node will not get included in the consensus process by other nodes unless the nodes that already participate in the consensus process choose to start listening to it. This means that the organizations controlling the nodes currently participating in the consensus process have control over which/whether new nodes get added.

I don't think there's anything wrong with or bad about this. But the gating of participation in the XRP ledger consensus process works in a materially different way than it does for the majority of cryptocurrencies and I think this is worth pointing out for the reader.

Let me know though if there's a specific way you'd like to rephrase this.

3:

This is not all ripple net does at all, that is not even the tip of the ice burg. EVEN ASSETS LIKE GOLD AND OIL COULD BE USED ON XCURRENT. Ripple net can be used as a liquidity source, an intermediary assets, a transaction facilitator, an inventory tracker, identification, healthcare. The possibilities are endless

I do mention gold here. We could use oil as an example if you think that would better help the reader envision Ripple's potential. I also mention XRP's application as a bridge currency to be used for liquidity.

As far as identification and healthcare are concerned, I haven't seen any material that's touched on Ripple's potential application in those spaces. I could envision arbitrary financial instruments being traded on ripple net and I could see how that could be useful in the context of healthcare. Identification I'm a little more confused about. Do you have any sources you could provide here? Feel free to suggest edits to the content too that better convey Ripple's potential to the reader.

Thanks again for taking the time to write up your thoughts on this and just a reminder to respond here if you want to continue the conversation.
